python/CHANGELOG.rst

```diff
@@ -12,12 +12,22 @@
   associated with each individual as a numpy array.
   (:user:`benjeffery`, :pr:`3153`)
 
+- Add ``TreeSequence.map_to_vcf_model`` method to return a mapping of
+  the tree sequence to the VCF model. 
+  (:user:`benjeffery`, :pr:`3163`)
 
 **Fixes**
 
 - Correct assertion message when tables are compared with metadata ignored.
   (:user:`benjeffery`, :pr:`3162`, :issue:`3161`)
   
+**Changes** 
+
+- ``TreeSequence.write_vcf`` now filters non-sample nodes from individuals
+  by default, instead of raising an error. These nodes can be included using the
+  new ``include_non_sample_nodes`` argument.
+  (:user:`benjeffery`, :pr:`3163`)
```

Comment:
We need to add a breaking change here: "By default individual names are now of the form "tsk_{individual.id}", whereas previously names (i.e., sample IDs in the VCF were always "``"tsk_{j}" for j in range(num_individuals)``. This may break some downstream code if individuals are specified. To fix, manually specify ``individual_names`` to the required pattern"

Reply:
Good point, fixed in b5de86fa